__tests__/app.test.js

```diff
@@ -35,4 +37,26 @@ describe('app()', () => {
       return request(app).get('/api/toothpicks').expect(404);
     });
   });
+  describe('/api', () => {
+    it('200: responds with status 200', () => {
+      return request(app).get('/api').expect(200);
+    });
+    it('200: should respond with a list of all available endpoints as a JSON object', () => {
+      return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({ body: { endpoints } }) => {
+          expect(typeof endpoints).toBe('object');
+          Object.values(endpoints).forEach((object) => {
+            expect(object).toHaveProperty('description', expect.any(String));
+            expect(object).toHaveProperty('queries', expect.any(Array));
+            expect(object).toHaveProperty(
+              'exampleResponse',
+              expect.any(Object)
+            );
+          });
+        });
+    });
+    it('400: return with a 400 error if a query is passed', () => {});
```

Comment:
Be sure not to add in anything you don't need yet. This looks like a test for another endpoint maybe?